src/utils/utilMethods.tsx

```diff
@@ -0,0 +1,8 @@
+// Generate random integer string
```

Comment:
this file is not getting used, would it be used in the future?

Reply:
I created this function when I was using another endpoint to manage posts (an incorrect endpoint) that needed a Post id. I'm not sure if we will use it in the future, but I will remove it from this PR.